source/includes/_contacts.md

```diff
@@ -129,7 +129,7 @@ contact_stage_id | Assign contact to this stage, if the contact does not yet exi
 present_raw_address | The address string for this contact, Apollo will intelligently infer the city, state, country, and time zone from your address | "San Francisco"
 
 
-## Update a Contact
+## Update a contact
```

Comment:
Casing update the description section on line no [121](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L121) and [122](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L122) to be consistent with [120](https://github.com/apolloio/apollo-api-docs/blob/80a972fb8380367506d3f84e6096153eca28fd2c/source/includes/_contacts.md?plain=1#L120)

`Last Name` -> `Last name`
`Company Name` -> `Company name`